doc/releases/v03.01.adoc

```diff
@@ -0,0 +1,33 @@
+= https://github.com/OpenSimulationInterface/open-simulation-interface/releases/tag/v3.1.0[v3.1.0 - OSI "Dynamic Duncan"]
+
+https://www.youtube.com/watch?v=fu3uA8K6ApQ>[(O SI)lent night]
+
+== Purpose
+
+This time of the year Santa comes with presents and has also thought about the sensor modelers of this planet. We introduce the current state of the ISO 23150 discussions including Ultrasonic and Camera Detections. The traffic signs have been reworked and the documentation has been extended for a perfect user experience. Trust me - it's amazing!
```

Comment:
Fixed linebreaks in context of PR #533 (9ac1d74d03234f5c351f4691c1accc995a9c33c8).